oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java

```diff
@@ -52,6 +52,7 @@
     private int superDatasetIndexShardsFactor = 5;
     private int indexRefreshInterval = 2;
     private int bulkActions = 2000;
+    private int syncBulkActions = 10000;
```

Comment:
Why put this in 10k? Is there any testing, which could prove this provides better performance?

Reply:
In our production environment, it's 100k. However, considering that most people try skywalking, the machine configuration is not so high, and this is a configurable item, so the initial value is set to 10k.

Reply:
I am a little concern about the size, whether it is too big for a normal user.